frontend/test/metabase/scenarios/smoketest/user.cy.spec.js

```diff
@@ -69,6 +69,8 @@ describe("smoketest > user", () => {
     });
     cy.button("Visualize").click();
 
+    cy.wait(1000); // slide-in animation for the table
```

Comment:
I don't fancy this, though there are already similar waits in this test file. Any better idea @nemanjaglumac ?

Reply:
Let me take a look. There has to be something else that we can wait for in the UI.
About those other arbitrary waits - we should get rid of them asap.

Reply:
@ariya done :)
https://github.com/metabase/metabase/pull/17056/commits/205c9e8f2441d5c98e61544e73dbee11e6348d77

Reply:
Thanks @nemanjaglumac :+1: